game.js

```diff
@@ -1,51 +1,86 @@
-const rls = require('readline-sync')
-
+const rls = ({ keyInYN } = require('readline-sync'));
+//restarted the fork because the first one won't let me push.
 /**
  * Starts the game by prompting the user if they want to play
  * Calls either gameLoop() or quitGame()
- * 
+ *
  * @returns {undefined}
  */
 const startGame = () => {
-
-}
+	if (rls.keyInYN('Hey want to play?')) {
+		console.log(`I have a random number in mind`);
+		console.log(`It's between 1 and 1000`);
+		console.log(`You have 10 guesses total`);
+		console.log('Want to play again?');
+		console.log("Let's start!");
+		gameLoop();
+	} else {
+		console.log('Have a nice life!');
+		quitGame();
+	}
+};
 
 /**
  * Logs "Goodbye!"
  * Calls process.exit() to quit the game
- * 
+ *
  * @returns {undefined}
  */
 const quitGame = () => {
-
-}
+	console.log('Goodbye!');
+	process.exit();
+};
 
 /**
  * Controls the flow of the game.
  * Should prompt the user to play again once all
  * guesses have been made or correct answer guessed
- * 
+ *
  * @returns {undefined}
  */
-const gameLoop = () => {
-
+const gameLoop = () => {};
```

Comment:
On the write track but needs to have some more code in here to get the user stories working. Think about while loops and generating the random number once then getting the guess.